### ISSUE TITLE
About the learning rate

### ISSUE THREAD

**Issue by czs1886** [potterhsu/easy-faster-rcnn.pytorch#26]
For the WarmUpMultiStepLR, if we maintain the default factor and warm up steps, the learning rate will  tend to be 0 after several steps since the lr is mulplicated by factor for number of steps times.

**Comment by Rajat-Mehta** [comment]
check this #7
